### PR TITLE
Modify existing alts for diff

### PR DESCRIPTION
**Pull request by isabela-pf**
# Welcome!

This is the working PR for the [Jupyter accessibility workshop alt text mini sprint](https://hopin.com/events/jupyter-accessibility-sprint-1/registration) (that's a lot of words)! This is where we collaborate during the event before we send a clean contribution over to the [main repo](https://github.com/jupyter/jupyter/).

For the full info about this event, please visit the [accessibility workshop planning repo](https://github.com/Quansight-Labs/jupyter-accessibility-workshops).

## How to contribute

⭐ For full info about how to interact with this PR, please visit the [How to Participate section of the agenda](https://github.com/Quansight-Labs/jupyter-accessibility-workshops/blob/main/events/2022-january-22/mentored_sprint_agenda.md#how-to-participate-in-writing-alt-text). Please contribute via [suggestions](https://hackmd.io/@marslee/BksnYfWVK) so we can easily review each other's work and give you credit in the commit history!

✏️ If you plan to work on a file, let people know by commenting below. You can do this as a group, and you can write alt text for as many images as you'd like. We can have multiple suggestions for the same alt text, but it would be nice if we spread out to cover more images.

📚 If you need a reference for how to format your suggestions, [alt text in Markdown files (ends with `.md`) follow this format](https://www.markdownguide.org/basic-syntax/#images-1) while [alt text in RST files (ends with `.rst`) follow this format](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/images.html).

🖼️ Images in this PR that already have alt text and ones that have none. 
- Images that appear in this PR with existing alt text are still worth reviewing and editing! If you think they could benefit from a different description, please suggest one.

❗ If you think an image is missing from these files, let @isabela-pf know and she'll add it to the PR.

## Images in this PR

Please refer to the `Files Changed Tab`

---

If you need anything, please ask @isabela-pf 🌻

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -318,46 +318,46 @@
 partners:
   - url: https://apple.com/
     src: apple.svg
-    alt: apple inc logo
+    alt: apple inc logo 
   - url: https://aws.amazon.com/
     src: aws.svg
-    alt: AWS logo
+    alt: AWS logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Amazon Web Services - Home Page
```

File: about.html [hunk 2]
```diff
@@ -318,46 +318,46 @@
...
-    alt: cal poly logo
+    alt: cal poly logo 
   - url: https://databricks.com/
     src: databricks.svg
-    alt: Databricks logo
+    alt: Databricks logo 
   - url: https://www.si.umich.edu/
     src: michigan.svg
-    alt: UMSI logo
+    alt: UMSI logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: University of Michigan School of Information - Home Page
```

File: about.html [hunk 3]
```diff
@@ -318,46 +318,46 @@
...
-    alt: apple inc logo
+    alt: apple inc logo 
   - url: https://aws.amazon.com/
     src: aws.svg
-    alt: AWS logo
+    alt: AWS logo 
   - url: https://www.berkeley.edu/
     src: berkeley-color.svg
-    alt: UC Berkeley logo
+    alt: UC Berkeley logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: University of California Berkeley - Home Page
```

File: _layouts/default.html [hunk 4]
```diff
@@ -71,7 +71,7 @@
                 <source media="(min-width: 768px)" srcset="/assets/logos/rectanglelogo-greytext-orangebody-greymoons.svg">
                 <img class="navbar-logo"
                      src="/assets/logos/rectanglelogo-greytext-orangebody-greymoons.svg"
-                     alt="Jupyter logo"
+                     alt="Jupyter logo "
```
Comment (ohrely) [inline _layouts/default.html]:
```suggestion
                     alt="Jupyter Home"
```

File: about.html [hunk 5]
```diff
@@ -282,34 +282,34 @@
 sponsors:
   - url: https://opendreamkit.org/
     src: EC-H2020.svg
-    alt: EU logo
+    alt: EU logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Horizon 2020 European Union funding for Research and Innovation logo - OpenDreamkit funded project Home Page
```

File: _layouts/default.html [hunk 6]
```diff
@@ -150,7 +150,7 @@ <h2>Follow us</h2>
       </section>
       <section class="footer-text">
         <div class="content content--wide">
-          <img src="/assets/logos/rectanglelogo-blacktext-blackbody-blackplanets.svg" width="189" height="51" alt="project jupyter logo">
+          <img src="/assets/logos/rectanglelogo-blacktext-blackbody-blackplanets.svg" width="189" height="51" alt="project jupyter logo ">
```
Comment (ohrely) [inline _layouts/default.html]:
```suggestion
          <img src="/assets/logos/rectanglelogo-blacktext-blackbody-blackplanets.svg" width="189" height="51" alt="Project Jupyter logo">
```

File: _layouts/default.html (hunk 4 above)
Comment (peytondmurray) [inline _layouts/default.html]: I'm working on this!
```suggestion
                     alt="Jupyter logo - Home Page"
```

File: about.html [hunk 7]
```diff
@@ -282,34 +282,34 @@
 sponsors:
   - url: https://opendreamkit.org/
     src: EC-H2020.svg
-    alt: EU logo
+    alt: EU logo 
   - url: https://www.fastly.com/
     src: fastly.svg
-    alt: fastly logo
+    alt: fastly logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Fastly - Home Page
```

File: about.html [hunk 8]
```diff
@@ -282,34 +282,34 @@
...
-    alt: EU logo
+    alt: EU logo 
   - url: https://www.fastly.com/
     src: fastly.svg
-    alt: fastly logo
+    alt: fastly logo 
   - url: https://www.google.com/
     src: google-color.svg
-    alt: google logo
+    alt: google logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Google - Home Page
```

File: widgets.html [hunk 9]
```diff
@@ -98,10 +98,10 @@ <h3 class="col-sm-12 section-header">Notebook Widgets</h3>
             <span class="gallery-title">ipyleaflet</span>
               <span>
                 <a href="https://mybinder.org/v2/gh/jupyter-widgets/ipyleaflet/stable?filepath=examples">
-                <img class="img-scaling" src="assets/widgets/mybinder.svg" alt="Binder">
+                <img class="img-scaling" src="assets/widgets/mybinder.svg" alt="Binder ">
                 </a>
                 <a href="https://github.com/jupyter-widgets/ipyleaflet">
-                <img class="img-scaling" src="assets/widgets/github.svg" alt="GitHub">
+                <img class="img-scaling" src="assets/widgets/github.svg" alt="GitHub ">
```
Comment (ohrely) [inline widgets.html]:
```suggestion
                <img class="img-scaling" src="assets/widgets/github.svg" alt="ipyleaflet GitHub Repo">
```

File: _layouts/default.html (hunk 6 above)
Comment (peytondmurray) [inline _layouts/default.html]:
```suggestion
          <img src="/assets/logos/rectanglelogo-blacktext-blackbody-blackplanets.svg" width="189" height="51" alt="Project Jupyter logo">
```

File: about.html [hunk 10]
```diff
@@ -282,34 +282,34 @@
...
-    alt: fastly logo
+    alt: fastly logo 
   - url: https://www.google.com/
     src: google-color.svg
-    alt: google logo
+    alt: google logo 
   - url: https://helmsleytrust.org
     src: helmsley.svg
-    alt: helmsley trust logo
+    alt: helmsley trust logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: The Leona M. and Harry B. Helmsley charitable trust - Home Page
```

File: about.html [hunk 11]
```diff
@@ -282,34 +282,34 @@
...
-    alt: google logo
+    alt: google logo 
   - url: https://helmsleytrust.org
     src: helmsley.svg
-    alt: helmsley trust logo
+    alt: helmsley trust logo 
   - url: https://www.microsoft.com/
     src: microsoft-color.svg
-    alt: microsoft logo
+    alt: microsoft logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Microsoft - Home Page
```

File: about.html [hunk 12]
```diff
@@ -282,34 +282,34 @@
...
-    alt: helmsley trust logo
+    alt: helmsley trust logo 
   - url: https://www.microsoft.com/
     src: microsoft-color.svg
-    alt: microsoft logo
+    alt: microsoft logo 
   - url: https://www.moore.org/
     src: moore.svg
-    alt: moore foundation logo
+    alt: moore foundation logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Gordon and Betty Moore foundation - Home Page
```

File: widgets.html [hunk 13]
```diff
@@ -98,10 +98,10 @@ <h3 class="col-sm-12 section-header">Notebook Widgets</h3>
             <span class="gallery-title">ipyleaflet</span>
               <span>
                 <a href="https://mybinder.org/v2/gh/jupyter-widgets/ipyleaflet/stable?filepath=examples">
-                <img class="img-scaling" src="assets/widgets/mybinder.svg" alt="Binder">
+                <img class="img-scaling" src="assets/widgets/mybinder.svg" alt="Binder ">
```
Comment (ohrely) [inline widgets.html]:
```suggestion
                <img class="img-scaling" src="assets/widgets/mybinder.svg" alt="ipyleaflet Examples Binder">
```

File: about.html [hunk 14]
```diff
@@ -282,34 +282,34 @@
...
-    alt: microsoft logo
+    alt: microsoft logo 
   - url: https://www.moore.org/
     src: moore.svg
-    alt: moore foundation logo
+    alt: moore foundation logo 
   - url: https://www.quansight.com/
     src: quansight.svg
-    alt: Quansight logo
+    alt: Quansight logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Quansight - Home Page
```

File: about.html [hunk 15]
```diff
@@ -282,34 +282,34 @@
...
-    alt: moore foundation logo
+    alt: moore foundation logo 
   - url: https://www.quansight.com/
     src: quansight.svg
-    alt: Quansight logo
+    alt: Quansight logo 
   - url: https://developer.rackspace.com/
     src: rackspace-color.svg
-    alt: rackspace logo
+    alt: rackspace logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Rackspace technology - Developer solutions documentation Page
```

File: about.html [hunk 16]
```diff
@@ -282,34 +282,34 @@
...
-    alt: Quansight logo
+    alt: Quansight logo 
   - url: https://developer.rackspace.com/
     src: rackspace-color.svg
-    alt: rackspace logo
+    alt: rackspace logo 
   - url: https://schmidtfutures.com/
     src: schmidt.svg
-    alt: schmidt futures logo
+    alt: schmidt futures logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Schmidt futures - Home Page
```

File: about.html [hunk 17]
```diff
@@ -282,34 +282,34 @@
...
-    alt: rackspace logo
+    alt: rackspace logo 
   - url: https://schmidtfutures.com/
     src: schmidt.svg
-    alt: schmidt futures logo
+    alt: schmidt futures logo 
   - url: https://sloan.org/
     src: alfred.svg
-    alt: sloan logo
+    alt: sloan logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Alfred P. Sloan foundation - Home Page
```

File: about.html [hunk 18]
```diff
@@ -318,46 +318,46 @@
 partners:
   - url: https://apple.com/
     src: apple.svg
-    alt: apple inc logo
+    alt: apple inc logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Apple Inc - Home Page
```

File: about.html [hunk 19]
```diff
@@ -318,46 +318,46 @@
...
-    alt: AWS logo
+    alt: AWS logo 
   - url: https://www.berkeley.edu/
     src: berkeley-color.svg
-    alt: UC Berkeley logo
+    alt: UC Berkeley logo 
   - url: https://www.bloomberg.com/
     src: bloomberg-color.svg
-    alt: bloomberg logo
+    alt: bloomberg logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Bloomberg - Home Page 
```

File: about.html [hunk 20]
```diff
@@ -318,46 +318,46 @@
...
-    alt: UC Berkeley logo
+    alt: UC Berkeley logo 
   - url: https://www.bloomberg.com/
     src: bloomberg-color.svg
-    alt: bloomberg logo
+    alt: bloomberg logo 
   - url: https://www.calpoly.edu/
     src: poly-color.svg
-    alt: cal poly logo
+    alt: cal poly logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: CalPoly - Home Page
```

File: about.html [hunk 21]
```diff
@@ -318,46 +318,46 @@
...
-    alt: bloomberg logo
+    alt: bloomberg logo 
   - url: https://www.calpoly.edu/
     src: poly-color.svg
-    alt: cal poly logo
+    alt: cal poly logo 
   - url: https://databricks.com/
     src: databricks.svg
-    alt: Databricks logo
+    alt: Databricks logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Databricks - Home Page
```

File: about.html [hunk 22]
```diff
@@ -318,46 +318,46 @@
...
-    alt: Databricks logo
+    alt: Databricks logo 
   - url: https://www.si.umich.edu/
     src: michigan.svg
-    alt: UMSI logo
+    alt: UMSI logo 
   - url: https://www.mongodb.com/
     src: mongodb.svg
-    alt: MongoDB logo
+    alt: MongoDB logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: MongoDB - Home Page
```

File: about.html [hunk 23]
```diff
@@ -318,46 +318,46 @@
...
-    alt: UMSI logo
+    alt: UMSI logo 
   - url: https://www.mongodb.com/
     src: mongodb.svg
-    alt: MongoDB logo
+    alt: MongoDB logo 
   - url: https://netflixtechblog.com/
     src: netflix-color.svg
-    alt: netflix logo
+    alt: netflix logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Netflix - Tech blog Home Page
```

File: about.html [hunk 24]
```diff
@@ -318,46 +318,46 @@
...
-    alt: MongoDB logo
+    alt: MongoDB logo 
   - url: https://netflixtechblog.com/
     src: netflix-color.svg
-    alt: netflix logo
+    alt: netflix logo 
   - url: https://www.quansight.com/
     src: quansight.svg
-    alt: Quansight logo
+    alt: Quansight logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Quansight - Home Page
```

File: about.html [hunk 25]
```diff
@@ -318,46 +318,46 @@
...
-    alt: netflix logo
+    alt: netflix logo 
   - url: https://www.quansight.com/
     src: quansight.svg
-    alt: Quansight logo
+    alt: Quansight logo 
   - url: https://quantstack.net/
     src: quantstack-color.svg
-    alt: QuantStack logo
+    alt: QuantStack logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: QuantStack - Home Page
```

File: about.html [hunk 26]
```diff
@@ -318,46 +318,46 @@
...
-    alt: Quansight logo
+    alt: Quansight logo 
   - url: https://quantstack.net/
     src: quantstack-color.svg
-    alt: QuantStack logo
+    alt: QuantStack logo 
   - url: https://www.simula.no/
     src: simula.svg
-    alt: Simula logo
+    alt: Simula logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Simula - Home Page
```

File: about.html [hunk 27]
```diff
@@ -318,46 +318,46 @@
...
-    alt: QuantStack logo
+    alt: QuantStack logo 
   - url: https://www.simula.no/
     src: simula.svg
-    alt: Simula logo
+    alt: Simula logo 
   - url: https://www.twosigma.com/
     src: two-sigma.svg
-    alt: Two Sigma logo
+    alt: Two Sigma logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: Two Sigma - Home Page
```

File: about.html [hunk 28]
```diff
@@ -318,46 +318,46 @@
...
-    alt: Simula logo
+    alt: Simula logo 
   - url: https://www.twosigma.com/
     src: two-sigma.svg
-    alt: Two Sigma logo
+    alt: Two Sigma logo 
   - url: https://2i2c.org/
     src: 2i2c.svg
-    alt: 2i2c logo
+    alt: 2i2c logo
```
Comment (trallard) [inline about.html]:
```suggestion
    alt: 2i2c.org - Home Page
```

File: about.html (hunk 23 above)
Comment (franzenr) [inline about.html]:
```suggestion
    alt: Netflix Tech Blog - Home Page 
```

File: about.html (hunk 28 above)
Comment (franzenr) [inline about.html]:
```suggestion
    alt: 2i2c.org Home Page 
```

File: about.html [hunk 29]
```diff
@@ -568,7 +568,7 @@ <h2 class="text-center">{{ page.donate_box.headline }}</h2>
                  width=300
                  height=100
                  loading="lazy"
-                 alt="numfocus logo">
+                 alt="numfocus logo ">
```
Comment (trallard) [inline about.html]:
```suggestion
                 alt="Numfocus - Home Page ">
```

**Comment by isabela-pf** [comment]
As mentioned in the workshop, I will leave this PR open for a week so you may revisit it and make changes. After that, I will commit suggestions and make a PR to the main repo. 

I'm so impressed that this much alt text was written in just over an hour! Y'all inspire me. 🌷

**Comment by isabela-pf** [comment]
I forgot to post the contributing PR! This work has been merged at [jupyter/jupyter.github.io #680](https://github.com/jupyter/jupyter.github.io/pull/680). Thank you all again for your contributions!

**Pull request closed without merging**